pybamm/models/submodels/particle/base_particle.py

```diff
@@ -26,12 +26,16 @@ def _get_standard_concentration_variables(self, c_s, c_s_xav):
         c_s_surf = pybamm.surf(c_s, set_domain=True)
 
         c_s_surf_av = pybamm.x_average(c_s_surf)
+        geo_param = pybamm.geometric_parameters
 
         if self.domain == "Negative":
             c_scale = self.param.c_n_max
+            active_volume = geo_param.a_n_dim * geo_param.R_n / 3
         elif self.domain == "Positive":
             c_scale = self.param.c_p_max
-
+            active_volume = geo_param.a_p_dim * geo_param.R_p / 3
+        c_s_r_av = pybamm.r_average(c_s_xav)
+        c_s_r_av_vol = active_volume * c_s_r_av * c_scale
```

Comment:
maybe it would be better here to define `c_s_r_av_vol = active_volume * c_s_r_av`, then times by `c_scale` in the variable dict to get the dimensional version? just to stick with the convention of the variables being non-dimensional?